x/auth/ante/unorderedtx/manager.go

```diff
@@ -8,19 +8,18 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"maps"
 	"os"
 	"path/filepath"
-	"sort"
+	"slices"
 	"sync"
 	"time"
-
-	"golang.org/x/exp/maps"
 )
 
 const (
-	// DefaultMaxUnOrderedTTL defines the default maximum TTL an un-ordered transaction
+	// DefaultmaxTimeoutDuration defines the default maximum duration an un-ordered transaction
 	// can set.
-	DefaultMaxUnOrderedTTL = 1024
+	DefaultMaxTimeoutDuration = time.Minute * 40
```

Comment:
if this was from audit - we should reference

Reply:
i wasn't actually able to find anything related to this in the 52 final audit. thats the only one i know of. maybe they suggested elsewhere.